### DISCUSSION TITLE
Vercel not creating deployments pointing to production domain

### DISCUSSION THREAD

**Discussion by abhyudayasrinet** [vercel/vercel#6161] [Help] [answered]
The project is connected to a Git Repository with the production branch set to `prod`

All had been working fine, until yesterday, it was noticed that our deployments to production were not being deployed to our connected domain (domain.com). The domain is assigned correctly to the `prod` branch on git.

However, I see the deployments not mentioning the domain (domain.com) for deployment. The last successful deployment from 5 days ago does show "domain.com" under domains in the overview section of the project.

Clicking on "Promote to Production" from the deployment details page of commit that is pushed on `prod` says "there is no production domain assigned."

While investigating this issue, I did find that there was a default domain (`<some text>.vercel.app`) with a label "Production" in the domains page (I guess this was the default domain from when the project was created).

I assumed deleting it might shift the production label to the correct domain (i.e. domain.com), but that didn't happen.

This is an urgent issue since it is affecting our production and we're in the middle of a launch.. :( 
Please let me know if you need more information.

**Comment by abhyudayasrinet** [comment, marked as answer]
changing the "Git branch" value under Settings->Domains->Edit Domain from `prod` to nothing fixed this.

**Comment by Mojtaba-amirii** [comment]
Quick fix > go to "Setting" > "Git"  and disable "Auto-assign Custom Production Domains" button and manualy promote
